_posts/2017/2017-02-23-why-i-love-working-as-a-scrum-master.md

```diff
@@ -0,0 +1,136 @@
+---
```

Comment:
@BirgitBader I sent Serena the post as she asked me to do so. Ann-Katrin has not reviewed it yet.